html/simple_ui/themes/nano/fonts/SIL-OFL-1.1-LICENSE.md

```diff
@@ -0,0 +1,92 @@
+This license governs the Font Awesome font files located in tgui/assets/fonts
```

Comment:
You should probably change that path.

Reply:
It was a copypaste of a folder, probably gonna remove it and cover the licensing in the README.md in the module folder.